templates/revoke.sh.j2

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+
+CADIR="{{ openvpn_key_dir }}"
+
+cd ${CADIR}
+
+gen_crl () {
+  # regenerate the certificate revocation list
+  openssl ca -gencrl -config ca.conf -out ca-crl.pem
```

Comment:
I also thought about that, but didn't know how to implement. After revoking each of the certificates in the list the crl file needs to be regenerated i.e.

openssl ca -gencrl -config ca.conf -out ca-crl.pem

needs to be run once in the end.